Formula/minimal-racket.rb

```diff
@@ -2,8 +2,9 @@ class MinimalRacket < Formula
   desc "Modern programming language in the Lisp/Scheme family"
   homepage "https://racket-lang.org/"
   url "https://mirror.racket-lang.org/installers/7.9/racket-minimal-7.9-src-builtpkgs.tgz"
-  sha256 "293aa8ef709a6240472f16833351ba66a9e461261d2813c7fb1cc5ddf59c3000"
+  sha256 "85b201aebc1ad1ec98ac590e18052d7ef8a81af280244d00ca1c28e8543b3fe9"
   license any_of: ["MIT", "Apache-2.0"]
+  revision 1
```

Comment:
Aside from the `sha256`, you only update the test, so this revision bump isn't necessary.

Reply:
It's because the upstream said the `sha256` changes is caused by their ubuntu PPA package changes, not the codes, so we should not bump the revision?

Reply:
Wait, so they kept the version the same but also changed the source code in a meaningful way? That seems odd. They should've just released a new version. I guess the revision bump is fine if this is in fact the case.

Reply:
See https://github.com/racket/racket/issues/3639 linked before

Reply:
racket is using x.y style version instead of semver or something alike. I think they can change the version strategy easily or quickly.